tox.ini

```diff
@@ -20,6 +20,7 @@ envlist = fix_lint,
 skip_missing_interpreters = true
 
 [testenv]
+allowlist_externals = bash
```

Comment:
Why is bash used here? We shouldn't need this

Reply:
It disables a warning when running tox.

Reply:
After removing this the CI fails, so I readded it. Let's remove this line in your PR together with the script